packages/core/src/payment/instrument/instrument-response-body.ts

```diff
@@ -35,6 +35,7 @@ export interface CardInternalInstrument extends BaseInternalInstrument {
     iin: string;
     last_4: string;
     method_type: 'card';
+    untrusted_shipping_address_card_verification_mode: string;
```

Comment:
Why not using the type enum here?
```suggestion
    untrusted_shipping_address_card_verification_mode: UntrustedShippingCardVerificationType;
```

Reply:
It is a response body object, we should use string to reflect what we received.

Reply:
Updated, since it is simpler to use enum here